src/lib/analyses.test.ts

```diff
@@ -752,11 +758,12 @@ describe('getExperimentAnalysesHealthIndicators', () => {
           "indication": Object {
             "code": "very high",
             "reason": "1.5 < x ≤ ∞",
-            "severity": "Error",
+            "recommendation": "Results are very imprecise, be careful about drawing conclusions. Extend for more precision",
```

Comment:
```suggestion
            "recommendation": "Results are very imprecise, be careful about drawing conclusions. Continue running the experiment for more precision",
```

maybe? I just feel like "extend" is vague... this is really up to you though.